### PR TITLE
ros: cmakelist update and ir-depth node publishing

### PR DESCRIPTION
**Pull request by rbudai98**
Signed-off-by: rbudai98 <robert.budai@analog.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ For more details about the SDK check the links below:
 | first-frame | <a href="https://github.com/analogdevicesinc/aditof-sdk-rework/tree/master/examples/first-frame"> C++ </a> <br> <a href="https://github.com/analogdevicesinc/aditof-sdk-rework/tree/master/bindings/python/examples/first_frame"> Python </a> | An example code that shows the steps required to get to the point where camera frames can be captured. |
 | first-frame-network | <a href="https://github.com/analogdevicesinc/aditof-sdk-rework/tree/master/examples/first-frame-network"> C++ </a> | An application that shows how to use the aditof sdk to talk to a remote ToF camera over the network. |
 | low_level_example | <a href="https://github.com/analogdevicesinc/aditof-sdk-rework/tree/master/bindings/python/examples/low_level_example"> Python</a> | A simple example of how to get access to the low-level API of the camera. |
+| ROS over network | <a href="https://github.com/analogdevicesinc/aditof-sdk-rework/tree/master/bindings/ros/"> Python</a> | ROS binding that publishes topics over network |
```
Comment (dNechita) [inline README.md:49]: Isn't the language supposed to be C++? (Instead of Python?)

File: bindings/ros/aditof_roscpp/CMakeLists.txt [hunk 2]
```diff
@@ -33,6 +33,7 @@ target_link_libraries(aditof_camera_node aditof::aditof)
 target_link_libraries(aditof_camera_node ${catkin_LIBRARIES})
 add_dependencies(aditof_camera_node ${PROJECT_NAME}_gencfg)
 
+#[[
```
Comment (dNechita) [inline bindings/ros/aditof_roscpp/CMakeLists.txt:36]: ?

File: README.md (hunk 1 above)
Comment (rbudai98) [reply README.md]: solved

File: bindings/ros/aditof_roscpp/CMakeLists.txt (hunk 2 above)
Comment (rbudai98) [reply bindings/ros/aditof_roscpp/CMakeLists.txt]: deleted

**Review by dNechita — APPROVED** [review]

**Pull request merged**